### DISCUSSION TITLE
Change the expiration date on an API key?

### DISCUSSION THREAD

**Discussion by ABHIMUCH** [deepgram/community#869] [General help]
Can I change when an existing API key is set to expire?

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jkroll-deepgram** [comment]
Hi @ABHIMUCH, you can't modify an API key's expiration date. The only way to modify an existing API key is to delete it, and then you can create a new one with the preferred expiration date.
